tests/integration/test_model.py

```diff
@@ -131,18 +131,6 @@
 
 @base.bootstrapped
 @pytest.mark.asyncio
-async def test_explicit_loop(event_loop):
```

Comment:
Can you explain why this test is invalid?

Reply:
The error ("Cannot run the event loop while another loop is running") occurs because you're not actually supposed to have multiple loops running in a single thread.  It wasn't explicitly checked for until 3.5.3, but we should not have been doing this in the first place.